torch/_torch_docs.py

```diff
@@ -3922,6 +3922,35 @@ def merge_dicts(*dicts):
     tensor(7.5107e-16, dtype=torch.float64)
 """.format(**common_args))
 
+add_docstr(torch.isin, r"""
+isin(elements, test_elements, assume_unique=False, invert=False) -> Tensor
+
+Tests if each element of :attr:`elements` is in :attr:`test_elements`. Returns
+a boolean tensor of the same shape as :attr:`elements` that is True for elements
+in :attr:`test_elements` and False otherwise.
+
+.. note::
+    One of :attr:`elements` or :attr:`test_elements` can be a scalar, but not both.
+
+Args:
+    elements (Tensor or Scalar): Input elements
+    test_elements (Tensor or Scalar): Values against which to test for each input element
+    assume_unique (bool, optional): If True, assumes both :attr:`elements` and
+        :attr:`test_elements` contain unique elements, which can speed up the
+        calculation. Default: False
+    invert (bool, optional): If True, inverts the boolean return tensor, resulting in True
+        values for elements _not_ in :attr:`test_elements`. Default: False
+
+Returns:
```

Comment:
This probably doesn't need a "Returns" section of the docs. It's typically only used when returning a (named) tuple.

Reply:
Unfortunately the docs are such a mess that's true. To rephrase: we should get in the habit of using "Returns" when a function returns a named tuple or similar datatstructure, but not when the return is clear from the description of the operation. We're inconsistent enough about it that you can leave it in if you like, however.

Reply:
This is just my opinion, but I'd prefer the "Returns" section being present unconditionally for consistency and because I find it convenient for identifying what I'm getting back from a quick glance at the documentation.

Reply:
That is a pretty opinion, though...

Let's follow-up offline.